packages/react/src/generators/application/files/material-ui/src/app/__fileName__.tsx__tmpl__

```diff
@@ -0,0 +1,102 @@
+<% if (classComponent) { %>
```

Comment:
This template could likely use some cleanup appearance wise since I removed the existing styles and only wrapped with a Container.